### PR TITLE
Closes #90 Feat/different fixes

### LINKED ISSUE #90: Different fixes
Look at the milestone pdf for the full description
- Colours
- Percentage Match Graphic
- Phone Numbers
- Support Contact Details

### PR DESCRIPTION
**Pull request by oskvortsov-bmt**
Closes #90 Feat/different fixes:
- colors
- adding percentage match graphic
- change support number

### CODE REVIEW

File: src/app/shared/components/candidate-tile/candidate-tile.component.scss [hunk 1]
```diff
@@ -42,18 +42,151 @@ h4 {
 
 
 .green-code-text {
-  color: #02af30 !important;
+  color: #36b37e !important;
```
Comment (zaikin-andrew) [inline src/app/shared/components/candidate-tile/candidate-tile.component.scss]: move all colors to __variables.scss_
**Just colors, not classes**

File: src/app/shared/components/candidate-tile/candidate-tile.component.scss [hunk 2]
```diff
@@ -42,18 +42,151 @@ h4 {
...
+$circle-width: 0.12em;
+
+// classes 2 extend
+.rect-auto{
+  clip: rect(auto, auto, auto, auto);
+}
+
+.pie {
+  position: absolute;
+  border: $circle-width solid red;
```
Comment (zaikin-andrew) [inline src/app/shared/components/candidate-tile/candidate-tile.component.scss]: wrong color. use variables with general colors. Color should match the color code according to the number of %.

File: src/app/shared/components/candidate-tile/candidate-tile.component.scss [hunk 3]
```diff
@@ -42,18 +42,151 @@ h4 {
...
+}
+
+.pie-fill {
+  transform: rotate(-180deg);
+}
+
+
+
+// main
+.c100 {
```
Comment (zaikin-andrew) [inline src/app/shared/components/candidate-tile/candidate-tile.component.scss]: What is `c100`? Change to an understandable name.

File: src/app/shared/components/job-tile/job-tile.component.scss [hunk 4]
```diff
@@ -1,3 +1,5 @@
+$purple: #8777D9;
```
Comment (zaikin-andrew) [inline src/app/shared/components/job-tile/job-tile.component.scss]: All colors move to the _variables.scss

File: src/app/shared/components/job-tile/job-tile.component.scss [hunk 5]
```diff
@@ -12,7 +14,12 @@
   justify-content: center;
   padding: .25rem .5rem;
   border-radius: 0.25rem;
-  background-color: #02af30;
+  background-color: #36b37e;
```
Comment (zaikin-andrew) [inline src/app/shared/components/job-tile/job-tile.component.scss]: All colors move to the _variables.scss. In all places.

**Comment by oskvortsov-bmt** [comment]
via Hubstaff
User: Oleg Skvortsov

Project: Hirefy-client - https://app.hubstaff.com/projects/300615
Date Range: 02/15/18 - 02/15/18
Work session total: 0:46:28

Grand total: 0:46:28

**Pull request merged**